modules/tensor_mechanics/src/actions/TensorMechanicsAction.C

```diff
@@ -24,16 +24,18 @@ InputParameters validParams<TensorMechanicsAction>()
   params.addParam<NonlinearVariableName>("temp", "The temperature"); // Deprecated
   params.addParam<NonlinearVariableName>("temperature", "The temperature");
 
-  MooseEnum strainType("SMALL INCREMENTAL FINITE", "SMALL");
+  MooseEnum strainType("SMALL FINITE", "SMALL");
   params.addParam<MooseEnum>("strain", strainType, "Strain formulation");
+  params.addParam<bool>("incremental", "Use incremental or total strain");
```

Comment:
I'm a little torn about whether this should be an enum or a bool, but I think I like it the way you have it.